### DISCUSSION TITLE
Multichain FID registry (ex: Base)?

### DISCUSSION THREAD

**Discussion by gigamesh** [farcasterxyz/protocol#192] [Q&A]
I think it would unlock a lot of potential use cases for Farcaster-native token projects on Base if contracts there could verify FID owners signed messages. 

I'm assuming this isn't possible unless the FID registry is mirrored on Base. Is that correct? If so, is there any possibility of the registry getting mirrored there, or some other solution?

**Comment by varunsrin** [comment]
Mirroring should be possible - it's not high up on our priority list but can be implemented by any party you're willing to trust.

**Comment by TimDaub** [comment]
We also have this issue with Kiwi and IMO this a problem that we should throw to Optimism and their Superchain architecture. Everyone has this problem in some shape or form
